packages/experimental/Menu/src/Menu/useMenuContextValue.ts

```diff
@@ -2,5 +2,5 @@ import { MenuContextValue } from '../context/menuContext';
 import { MenuState } from './Menu.types';
 
 export const useMenuContextValue = (state: MenuState): MenuContextValue => {
-  return { open: state.open, setOpen: state.setOpen, triggerRef: state.triggerRef };
+  return { open: state.open, setOpen: state.setOpen, triggerRef: state.triggerRef, isSubmenu: state.isSubmenu };
```

Comment:
Q: why is this file called useMenuContext_Value_ ? Like why "value"?

Reply:
It's because it creates the value that is fed into the MenuContext Provider. I can rename it useCreateMenuContextValue or something like that, but since it's a hook it needs to start with the word "use"

Reply:
Nah I like this name better. Thanks for the explanation!